src/main/features/response/views/confirmation/full-admission.njk

```diff
@@ -11,9 +12,15 @@
   <p>{{ t('We’ve emailed {{ claimantName }} your offer to pay by {{ paymentDate }} and your explanation of why you can’t pay before then.',
```

Comment:
Please update all these if endif statements to be if elif endif for the payment options.